evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -1418,3 +1422,48 @@ def test_visualize_decision_trees(fitted_tree_estimators, tmpdir):
     src = visualize_decision_tree(estimator=est_reg, filled=True, max_depth=2)
     assert src.format == 'pdf'
     assert isinstance(src, graphviz.Source)
+
+
+def test_linear_coefficients_errors():
+    X = pd.DataFrame([[1, 2, 3, 5],
+                      [3, 5, 2, 1],
+                      [5, 2, 2, 2],
+                      [3, 2, 3, 3]])
+    y = pd.Series([2, 1, 3, 4])
+
+    dt = DecisionTreeRegressor()
+    dt.fit(X, y)
```

Comment:
Really nit-picky but to make it more obvious, can we mock the DT's fit method to show that we don't need it (and potentially save some time lolol)